openlibrary/accounts/model.py

```diff
@@ -19,6 +19,9 @@
 from openlibrary.core import lending, helpers as h
 
 
+def append_random_suffix(text, limit=9999):
+    return '%s_%s' % (text, random.randint(0, limit))
```

Comment:
Why the underscore? Most people are used to `YouRock19283`, not `YouRock_19283`.

Also, I would think each iteration of suffixing would append a single new random digit, rather than a new set of up to 4 random digits. So if `retries` is 5, then the max number of suffixed digits you'll get is 5, not 20.